### DISCUSSION TITLE
How can I turn off specific warnings?

### DISCUSSION THREAD

**Discussion by rempas** [Pure-D/serve-d#215] [Q&A] [answered]
Hi and thanks for your work with serve-d! I would like to know if there is a way to turn of specific warnings. I tried "serve-d --help" and have read the "README.md" but wasn't able to find how to do that. Am I missing something?

**Comment by WebFreak001** [comment, marked as answer]
If you want to toggle D-Scanner warnings in a cross-IDE way you can create a dscanner.ini file in your project. If you use vscode with code-d, create the file and run the command "Insert default dscanner.ini content" and then you can turn off/on the checks you want to change.

Additionally when you put your cursor on warnings you can run code actions (light bulbs) to ignore the line or ignore them in the entire project (inserts a vscode workspace setting in code-d)

**Reply by rempas** [reply]
Thank you! The "dscanner.ini" file should be in the root directory of my project right? Where can I find the configuration options?

**Reply by WebFreak001** [reply]
Use the `Insert default dscanner.ini content` command in code-d or run (if you have it installed) `dscanner --defaultConfig` in your project directory.

The description for each check appears next to to check itself as a comment.

**Reply by rempas** [reply]
Thanks a lot!
